Classes/ProcessManager/State/State.php

```diff
@@ -0,0 +1,215 @@
+<?php
+namespace Neos\Cqrs\ProcessManager\State;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+use Doctrine\ORM\Mapping as ORM;
+use TYPO3\Flow\Object\ObjectManagerInterface;
+
+/**
+ * Process Manager State
+ *
+ * All methods which change the data of this state will trigger automatic persistence. This is necessary in order to
+ * support rows of commands and events handled by a single process manager which are executed synchronously within
+ * a single PHP request.
+ *
+ * @Flow\Entity
+ */
+class State
```

Comment:
Yes, I actually named it `ProcessState` at some point, don't know why it got lost.